industrial_ci/src/builders/colcon.sh

```diff
@@ -34,7 +34,12 @@ function builder_run_build {
 function builder_run_tests {
     local extend=$1; shift
     local ws=$1; shift
-    ici_exec_in_workspace "$extend" "$ws" colcon test --event-handlers "${_colcon_event_handlers[@]}" console_cohesion+
+    if [ "$IMMEDIATE_TEST_OUTPUT" == true ]; then
+        _colcon_event_handlers+=("console_direct+")
```

Comment:
This variable is used by subsequent builds as well and should not be modified.

Reply:
Yep sry did not notice.